python/ray/actor.py

```diff
@@ -668,6 +671,12 @@ def __del__(self):
         # there are ANY handles in scope in the process that created the actor,
         # not just the first one.
         worker = ray.worker.get_global_worker()
+        if (worker.mode == ray.worker.SCRIPT_MODE
+                and self._ray_actor_driver_id.id() != worker.worker_id):
+            # If the worker is a driver and driver id has changed because
+            # Ray was shut down re-initialized, the actor is already cleaned up
+            # and we don't need to send `__ray_terminate__` again.
+            return
```

Comment:
Can we actually log a warning in this case, e.g., `logger.warn(...)`?

This should be pretty rare and it'd be useful to see that this code path is actually getting hit.

Reply:
done